Tests/SwiftParserTest/translated/ConsecutiveStatementsTests.swift

```diff
@@ -102,8 +102,8 @@ final class ConsecutiveStatementsTests: XCTestCase {
       }
       """,
       diagnostics: [
-        DiagnosticSpec(locationMarker: "1️⃣", message: "consecutive declarations on a line must be separated by ';'", fixIts: ["insert ';'"]),
-        DiagnosticSpec(locationMarker: "2️⃣", message: "consecutive declarations on a line must be separated by ';'", fixIts: ["insert ';'"]),
+        DiagnosticSpec(locationMarker: "1️⃣", message: "consecutive declarations on a line must be separated by ';'", fixIts: ["insert newline", "insert ';'"]),
+        DiagnosticSpec(locationMarker: "2️⃣", message: "consecutive declarations on a line must be separated by ';'", fixIts: ["insert newline", "insert ';'"]),
```

Comment:
IMO we should offer newline before `;` as that is most common and swift style to do. 

What do you think?

Reply:
the fixed source below only adds newline if the `insert newline` is added as the last. 
That will insert newline before the `;`

Reply:
> IMO we should offer newline before ; as that is most common and swift style to do.
> What do you think?

I agree

> the fixed source below only adds newline if the insert newline is added as the last.
> That will insert newline before the ;

I think what we should do is to change `FixItApplier.init(diagnostics: [Diagnostic], withMessages messages: [String]?)` to only apply the first Fix-It for each diagnostic, not all of them. If a diagnostic offers multiple Fix-Its they should be considered mutually exclusive.